src/baseIcons/icon-sign-language-content.svg

```diff
@@ -1 +1,7 @@
+    </g>
```

Comment:
Fill seems to have been changed to another blueish color. Check if this is wanted change.